### PR TITLE
mv customization overview to reference

### PR DESCRIPTION
**Pull request by johrstrom**
**Modify this link to include the branch name, and possibly the page this PR modifies**:

https://osc.github.io/ood-documentation-test/coalese-customize/

This moves the `customization overview` to reference and makes `customizations` a direct navigation link.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201981671630881) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by gbyrket — COMMENTED** [review]
Hey Jeff.  I could be missing something here, but your description states that you moved customization overview under reference.

What I'm seeing is that you moved customization up 1 level in folder structure, which appears to be part of the change requirements.

What I'm not seeing is that you put customization overview under reference.  I only see that you added pun-environment under reference.

If this was your intention, please add some clarity around it so I can approve this change.

**Review by Oglopf — APPROVED** [review]
I really like the configuration reference section. This looks good!

File: source/reference/pun-environment.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. _customization-overview:
+.. _pun-environment:
 
-Overview
-========
+Pun Environment
+===============
```
Comment (johrstrom) [inline source/reference/pun-environment.rst:4]: @gbsoftwaresolutions  Yes I renamed `Customization Overview` to `Pun Environment`. You can see at the top of this card the filename was changed.

I figure it is more specific to the toggling the PUN's environment and not a general overview.

**Comment by gbyrket** [comment]
> I really like the configuration reference section. This looks good!

Oops.  Sorry Travis.

File: source/reference/pun-environment.rst (hunk 1 above)
Comment (gbyrket) [reply source/reference/pun-environment.rst:4]: Cool. Thanks for the clarification.

**Comment by johrstrom** [comment]
This got jacked up during the rebase, I'm going to have to redo it.

**Pull request closed without merging**